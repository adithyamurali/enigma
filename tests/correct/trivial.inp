* B BETA I II III AAAA
Hello world
* B BETA III IV I BLEY
The Bombola faints in the hot Bowral tree
Where fierce Mullengudgerys smothering fires
Far from the breezes of Coolgardie
Burn ghastly and blue as the day expires
And Murriwillumba complaineth in song
For the garlanded bowers of Woolloomooloo
And the Ballarat Fly and the lone Wollongong
They dream of the gardens of Jamberoo
The wallabi sighs for the Murrubidgee
For the velvety sod of the Munno Parah
Where the waters of healing from Muloowurtie
Flow dim in the gloaming by Yaranyackah
The Koppio sorrows for lost Wolloway
And sigheth in secret for Murrurundi
The Whangeroo wombat lamenteth the day
That made him an exile from Jerrilderie
* B BETA III IV I AXLE
FROM his shoulder Hiawatha
Took the camera of rosewood
Made of sliding folding rosewood
Neatly put it all together
In its case it lay compactly
Folded into nearly nothing
But he opened out the hinges
Pushed and pulled the joints
   and hinges
Till it looked all squares
   and oblongs
Like a complicated figure
In the Second Book of Euclid

* B BETA III IV I AXLE
HYIHL BKOML IUYDC MPPSF SZW
SQCNJ EXNUO JYRZE KTCNB DGU
FLIIE GEPGR SJUJT CALGX SNCTM KUF
WMFCK WIPRY SODJC VCFYQ LV
QLMBY UQRIR XEPOV EUHFI RIF
KCGVS FPBGP KDRFY RTVMW GFU
NMXEH FHVPQ IDOAC GUIWG TNM
KVCKC FDZIO PYEVX NTBXY AHAO
BMQOP GTZX
VXQXO LEDRW YCMMW AONVU KQ
OUFAS RHACK
KXOMZ TDALH UNVXK PXBHA VQ
XVXEP UNUXT XYNIF FMDYJ VKH
* B BETA III II I AAAZ

a
aa
aaa
aaaa
aaaaa
aaaaaa
aaaaaaa
aaaaaaaa
aaaaaaaaa
aaaaaaaaaa
* B BETA VI III II AAAZ
Check tHAT     case and spacing       do NOT matter
There are blanks at the end of this line
     And at the beginning of this one
* B BETA III II I AAAZ
aaaaaaaaaaaaaaaaaaaaaaaaaaaa
* B BETA III I II AAAZ
aaaaaaaaaaaaaaaaaaaaaaaaaaaa
* C GAMMA V VI II VGHJ
The Phoenix and the Turtle
Let the bird of loudest lay
On the sole Arabian tree
Herald sad and trumpet be
To whose sound chaste wings obey
But thou shrieking harbinger
Foul precurrer of the fiend
Augur of the fevers end
To this troop come thou not near
From this session interdict
Every fowl of tyrant wing
Save the eagle featherd king
Keep the obsequy so strict
Let the priest in surplice white
That defunctive music can
Be the deathdivining swan
Lest the requiem lack his right
And thou trebledated crow
That thy sable gender makst
With the breath thou givst and takst
Mongst our mourners shalt thou go
Here the anthem doth commence
Love and constancy is dead
Phoenix and the turtle fled
In a mutual flame from hence
So they lovd as love in twain
Had the essence but in one
Two distincts division none
Number there in love was slain
Hearts remote yet not asunder
Distance and no space was seen
Twixt the turtle and his queen
But in them it were a wonder