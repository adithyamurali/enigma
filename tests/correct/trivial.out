ILBDA AMTAZ
LVMYD OOWCU SSDZV FDHFR FIMAY JRTIV ZFMK
AWSEX AKLEY PYZKA DIHWL MXGTI AKCMQ KLLQC CRPBL
TRJPG PDELS XDPRX KFVXJ RENYH XOGC
TNIZJ SRNAZ QFCFL TEXBK ZKQZK CJVDN CZU
KFHFA CZOBC MESQO IAAGO FIWJG QAWUA VFR
AEDAT KQOHE HQFMZ KYNMT AWZUY RWKFX UGYOG Q
TXRDG BQNNT IFOAO BBWKO IVKXV UVZMB YZFVV AU
LSSFR AIPYA YHBDN XPJWA TYQDI IYPVD D
HRNPG MBCZQ AKKUV EFESJ RCGHU YPJPZ WV
QIWWR GKSHA NPTXP RGGXF IQGQW RAQVW S
RUJVB MUNCE FSTBT TPSSY NVAXQ JGQJO BUJMD MGD
QWPNN UXVVZ JGUPL SODHI HILOV PQWXK PZL
QJZJP XQMTE AYIBI JRVCO FYISE YKKPP A
IRGOO USNWM TRZXE AZIZD KCSIO IIGRF J
FNQPX DRDIM CDECF IIANU CAWUG HDIWU EFH
IXDXU BGBQZ FIWPV XRXKA LIGJC SEYOH OBI
HYIHL BKOML IUYDC MPPSF SZW
SQCNJ EXNUO JYRZE KTCNB DGU
FLIIE GEPGR SJUJT CALGX SNCTM KUF
WMFCK WIPRY SODJC VCFYQ LV
QLMBY UQRIR XEPOV EUHFI RIF
KCGVS FPBGP KDRFY RTVMW GFU
NMXEH FHVPQ IDOAC GUIWG TNM
KVCKC FDZIO PYEVX NTBXY AHAO
BMQOP GTZX
VXQXO LEDRW YCMMW AONVU KQ
OUFAS RHACK
KXOMZ TDALH UNVXK PXBHA VQ
XVXEP UNUXT XYNIF FMDYJ VKH

FROMH ISSHO ULDER HIAWA THA
TOOKT HECAM ERAOF ROSEW OOD
MADEO FSLID INGFO LDING ROSEW OOD
NEATL YPUTI TALLT OGETH ER
INITS CASEI TLAYC OMPAC TLY
FOLDE DINTO NEARL YNOTH ING
BUTHE OPENE DOUTT HEHIN GES
PUSHE DANDP ULLED THEJO INTS
ANDHI NGES
TILLI TLOOK EDALL SQUAR ES
ANDOB LONGS
LIKEA COMPL ICATE DFIGU RE
INTHE SECON DBOOK OFEUC LID

N
FT
ZMG
ISXI
PJWGD
NJJCO Q
TYRIG DM
XFIES RWZ
GTOIU IEKK
DCSHT PYOEP
BGFIO YCOVH EMPIO JIUYW YRLZY DMMHG CGDS
WIDLX EDFJZ CUNCZ FMOHU YKXVN SVUZC TT
JCHQF MOQPU HXZZW KLIIU MWEIE H
NFTZM GISXI PJWGD NJJCO QTYRI GDM
JCNBE GNHEJ ZLEKN SCFKB KNQHS PBK
VKBZO QTWWT ZRUHV WYYHW VS
TPFQQ USLNJ RLEGR IZULU VA
WUFUV NIMVR MCNKS MDBCB
KGAQA ZWZBZ XRLLP XNBIS L
BSFKT IOZMV BLWCT JIGJB WNEWI BV
SNJNE CBCOI UTWUS BJVLU ZEHMV
ZZMTQ PWEWM WNKHZ YEKAA HIJ
MEQHB MTGXA OOOAK LBDH
MAAUY KBDAB LTCHW YGYRR QBSZG C
OZNWG UKCBD KEZSA JADMQ PPWJ
LQYFN METGY BYCDC DDBQB X
CPLJE EFKQE CAEVE OQWFS COMX
XJTKL SUDRJ AUFSG TXWYR RV
HJFDX SOWQC OQFFF LWCKJ NYQCT CQ
ZFMVJ KIJDA YAYXT FEMYS XG
KAEEF CAZUU ZJYKK FVDBL CL
ALOBH FUZYZ XMVFQ DSWQO QSZDU N
ZOMBF FCOHF TKVHO FJQIB XJ
ARSYE AQHFE BUIYR WRQWN YVF
DSJAE MVFLB OUOMO ESIWO OJYVR RMRQN
ABSZC HXBMC SMVGV JLVZM TLQBZ XSV
VACJC LBEHX VKPRI JYKCT XCSWX
ZSHWT AZKWE MHPUK RWBYB GA
ZZTJI MVHTH LWTIX NQCIY NFH
JOJDW XPBJZ JQLUP CSTMU FIW
CAONJ VYKKO TGJKX FRZDP QGW
CIPFB XRDBX SQCZI LUTZG A
CZJHL AERIF RBSOF KEHBD HPOX
EYBKO NIXFB ODTBE KLCSV VWBJA
ROJUG DJKQV YDVAG BDBPC AIKPM
ZAHED KSHLP UQAEM BLVHF NFGYK
CQJOK CCBZO KYIUK HMJPJ SBATS
DWMUW EKVAV SYVED XTLBI UB